From 2fd507b75661c8ec2b9be07e8eaf613bdaddb854 Mon Sep 17 00:00:00 2001
From: Hideki Yamane <henrich@debian.org>
Date: Mon, 25 Jan 2016 15:23:42 +0900
Subject: [PATCH] use lintian by default

It prevents more lintian error in archive, and will improve quality
of package
---
 debian/control | 3 ++-
 dput           | 2 +-
 dput.cf        | 2 +-
 3 files changed, 4 insertions(+), 3 deletions(-)

diff --git a/debian/control b/debian/control
index f16a1e6..b8f3019 100644
--- a/debian/control
+++ b/debian/control
@@ -10,7 +10,8 @@ Vcs-Browser: http://anonscm.debian.org/gitweb/?p=collab-maint/dput.git
 Package: dput
 Architecture: all
 Depends: python (>= 2.5), gnupg
-Suggests: openssh-client, lintian, mini-dinstall, rsync
+Recommends: lintian
+Suggests: openssh-client, mini-dinstall, rsync
 Description: Debian package upload tool
  dput allows you to put one or more Debian packages into the archive.  This
  package also includes a dcut utility which can be used to generate and / or
diff --git a/dput b/dput
index 8b6c105..2447138 100755
--- a/dput
+++ b/dput
@@ -101,7 +101,7 @@ def read_configs(extra_config, debug):
     config.set('DEFAULT', 'allow_dcut', '0')
     config.set('DEFAULT', 'distributions', '')
     config.set('DEFAULT', 'allowed_distributions', '')
-    config.set('DEFAULT', 'run_lintian', '0')
+    config.set('DEFAULT', 'run_lintian', '1')
     config.set('DEFAULT', 'run_dinstall', '0')
     config.set('DEFAULT', 'check_version', '0')
     config.set('DEFAULT', 'scp_compress', '0')
diff --git a/dput.cf b/dput.cf
index 9296262..b334fd3 100644
--- a/dput.cf
+++ b/dput.cf
@@ -7,7 +7,7 @@ method			= ftp
 hash			= md5
 allow_unsigned_uploads	= 0
 allow_dcut		= 0
-run_lintian		= 0
+run_lintian		= 1
 run_dinstall		= 0
 check_version		= 0
 scp_compress		= 0
-- 
2.7.0

